hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```
@@ -112,6 +112,7 @@ hedera.transaction.maxMemoUtf8Bytes=100
 hedera.transaction.maxValidDuration=180
 hedera.transaction.minValidDuration=15
 hedera.transaction.minValidityBufferSecs=10
+hedera.transaction.eip2930.enabled=false
```

We can set this to true now after offline clarification
```suggestion
hedera.transaction.eip2930.enabled=true
```

---

updated.